From ec2b935d2af33651f34de8bd6e428156005c2e3d Mon Sep 17 00:00:00 2001
From: Martin Wilck <mwilck@suse.com>
Date: Tue, 12 May 2020 22:13:51 +0200
Subject: [PATCH] multipath-tools Makefile: add install dependency

$(libdir) must exist before running "make install" on prioritizer, checker,
and foreign libraries.

Cc: Christian Hesse <mail@eworm.de>

Signed-off-by: Martin Wilck <mwilck@suse.com>
Signed-off-by: Alexander Egorenkov <egorenar-dev@posteo.net>
---
 Makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Makefile b/Makefile
index fec3b73b..8bcaba66 100644
--- a/Makefile
+++ b/Makefile
@@ -32,6 +32,10 @@ libmultipath libdmmp: libmpathcmd
 libmpathpersist multipath multipathd: libmultipath
 mpathpersist multipathd:  libmpathpersist
 
+libmultipath/checkers.install \
+	libmultipath/prioritizers.install \
+	libmultipath/foreign.install: libmultipath.install
+
 $(BUILDDIRS.clean):
 	$(MAKE) -C ${@:.clean=} clean
 
-- 
2.28.0

